NOTES.md

```diff
@@ -1,3 +1,132 @@
+# Release 2.1.0-beta (Forbidden Planet)
+
+-- issues up-to-date with the 2.1 milestone as of 2015-07-12 20:00 UTC
+-- contributors " " next as of 3dac994 and docs:master as of 0b661c4
+
+This is a beta release for RethinkDB 2.1. **It is not for production use and has known bugs.** Please do not use this version for production data.
+
+We are looking forward to your bug reports on GitHub (http://github.com/rethinkdb/rethinkdb/issues/new) or on our mailing list (https://groups.google.com/forum/#!forum/rethinkdb).
+
+Release highlight:
+
+* Automatic failover
```

Comment:
We might want to mention Raft here -- since that's likely the first question sophisticated users will ask.